modules/cli/src/integrationTest/java/org/apache/ignite/internal/cli/commands/recovery/partitions/reset/ItResetPartitionsTest.java

```diff
@@ -84,6 +81,11 @@ public void testResetPartitionZoneNotFound() {
 
     @Test
     public void testResetPartitionTableNotFound() {
+        if (enabledColocation()) {
```

Comment:
Perhaps, using `Assumptions.assumeFalse(enabledColocation());` is the right way to ignore the test when the colocation feature is enabled.

Reply:
done